src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  searchText: string;
+  setSearchText: (text: string) => void;
+  reset: () => void;
+  addMovie: (movie: Movie) => void;
+};
+
+const defaultPicture = (
```

Comment:
1. constants should start with uppercase
2. I suggest to move all constants into `src->constants` folder
```suggestion
const DEFAULT_PICTURE= (
```